purging nix and re-installing nix fails

This is the error I see :
```log
The Nix package manager was successfully installed.
See STARTING_GUIDE.md if you're new here.
Nix interpreter already on version: 2.14.1
❯ make run-clojure
yarn shadow-cljs watch mobile
Configuring Nix shell for target 'clojure'...
error:
       … while evaluating the attribute 'shells'

         at /Users/siddarthkumar/code/status-mobile/nix/default.nix:11:10:

           10| in {
           11|   inherit pkgs targets shells;
             |          ^
           12| }

       … while calling the 'import' builtin

         at /Users/siddarthkumar/code/status-mobile/nix/pkgs.nix:29:4:

           28|   # import nixpkgs with a config override
           29|   (import nixpkgsSrc) {
             |    ^
           30|     config = defaultConfig // config;

       (stack trace truncated; use '--show-trace' to show the full trace)

       error: file 'nixpkgs' was not found in the Nix search path (add it using $NIX_PATH or -I)

       at «none»:0: (source not available)
make: *** [run-clojure] Error 1
```

--- a/nix/pkgs.nix
+++ b/nix/pkgs.nix
@@ -3,20 +3,13 @@
 { config ? { } }:
 
 let
-  inherit (import <nixpkgs> { }) fetchFromGitHub;
-
   # For testing local version of nixpkgs
   #nixpkgsSrc = (import <nixpkgs> { }).lib.cleanSource "/home/jakubgs/work/nixpkgs";
 
   # We follow the master branch of official nixpkgs.
-  nixpkgsSrc = fetchFromGitHub {
-    name = "nixpkgs-source";
-    owner = "NixOS";
-    repo = "nixpkgs";
-    rev = "e7603eba51f2c7820c0a182c6bbb351181caa8e7";
-    sha256 = "sha256-XJZ/o17eOd2sEsGif+/MQBnfa2DKmndWgJyc7CWajFc=";
-    # To get the compressed Nix sha256, use:
-    # nix-prefetch-url --unpack https://github.com/${ORG}/nixpkgs/archive/${REV}.tar.gz
+  nixpkgsSrc = builtins.fetchTarball {
+    url = "https://github.com/NixOS/nixpkgs/archive/e7603eba51f2c7820c0a182c6bbb351181caa8e7.tar.gz";
+    sha256 = "sha256:0mwck8jyr74wh1b7g6nac1mxy6a0rkppz8n12andsffybsipz5jw";
   };
 
   # Status specific configuration defaults

